lib/Security/UserRegisterPolicyProvider.php

```diff
@@ -11,12 +11,15 @@
 use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Security\PolicyProvider\PolicyProviderInterface;
 
 /**
+ * @deprecated Deprecated since 2.1. No longer used. This policy was moved to the eZ/Publish/Core/settings/policies.yml.
+ *
  * Adds the user/register policy.
  */
 class UserRegisterPolicyProvider implements PolicyProviderInterface
 {
     public function addPolicies(ConfigBuilderInterface $configBuilder)
     {
+        @trigger_error('Method ' . __METHOD__ . ' is deprecated since 2.1', E_USER_DEPRECATED);
         $configBuilder->addConfig(['user' => ['register' => null]]);
```

Comment:
Maybe let's remove that line only to avoid at the same time users getting hit by missing class and messing up kernel policy matrix. Of course keep deprecation warning.
**But before taking any further action, let's sync on that tomorrow @mikadamczyk @webhdx**